src/App.tsx

```diff
@@ -1,61 +1,41 @@
+import React, { useState } from 'react';
+
 import './App.scss';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { TodoForm } from './components/TodoForm';
+import { getUser } from './utils';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+const todos: Todo[] = todosFromServer.map((todo) => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [updatedTodo, setUpdatedTodo] = useState(todos);
+
+  const addNewTodo = (todo: Todo) => {
```

Comment:
```suggestion
  const addNewTodo = (todo: Omit<Todo, 'id'>) => {
```

Reply:
and here we need to change parameter type since our parameter won't contain id
It is called [utility type](https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys)